taco-tuesday-rest-api/src/main/java/com/muskopf/tacotuesday/bl/repository/EmployeeRepository.java

```diff
@@ -10,5 +10,5 @@
     boolean existsEmployeeByFirstNameAndLastName(String firstName, String lastName);
     boolean existsEmployeeBySlackId(String slackId);
 
-    Employee findEmployeeBySlackId(String slackId);
+    Employee findBySlackId(String slackId);
```

Comment:
Does the same thing, looks nicer.
I'll propagate this to the rest of the JPA guys eventually.